packages/admin/cms-admin/src/dam/Table/FolderTableRow.tsx

```diff
@@ -176,6 +192,12 @@ export const FolderTableRow: React.FunctionComponent<FolderTableRowProps> = ({
         }
     }, [dragSource, dropTarget, dropTargetFile, dropTargetItem, rowRef]);
 
+    React.useEffect(() => {
+        if (scrollIntoView) {
+            rowRef.current?.scrollIntoView({ behavior: "smooth", block: "center" });
+        }
+    }, [rowRef, scrollIntoView]);
```

Comment:
Refs are not required in the dependency array

Reply:
It was added automatically by eslint. I guess because it wasn't created directly in the component but it originates from `useDropzone()`

```
 const { rootRef: rowRef } = useDropzone({
```

Therefore, I wasn't sure if it's still save to remove it